bin/travis-run.sh

```diff
@@ -11,4 +11,4 @@ trap "cleanup" EXIT
 # Check PEP-8 code style and McCabe complexity
 flake8 --statistics --show-source ckanext
 
-nosetests --ckan --nologcapture --with-pylons=subdir/test.ini --with-coverage --cover-package=ckanext.geocat --cover-inclusive --cover-erase --cover-tests ckanext/geocat
+nosetests --ckan --nocapture --nologcapture --with-pylons=subdir/test.ini --with-coverage --cover-package=ckanext.geocat --cover-inclusive --cover-erase --cover-tests ckanext/geocat
```

Comment:
This was just an adaptation to making it easier to debug the tests on Travis.

Reply:
I guess when we agree on the tests that you implemented here, it would be nice to remove `--nocapture` since this will all the time immediately print output of any tests.